app/views/catalog/_zero_results.html.erb

```diff
@@ -1,12 +1,12 @@
 <h2><%= t 'blacklight.search.zero_results.title' %></h2>
 <div id="documents" class="noresults">
-  <h3 class='modify-search'> <%= t 'blacklight.search.zero_results.modify_search' %></h3>
+  <h3 class='modify-search'><%= blacklight_icon 'earthworks-search' %> <%= t 'blacklight.search.zero_results.modify_search' %></h3>
```

Comment:
I tried to use blacklight's search icon, but. could not get it to resize at all.  If somebody knows what I was doing wrong, happy to yank this icon for the Blacklight provided one.